Extension/src/LanguageServer/configurations.ts

```diff
@@ -2050,13 +2067,33 @@ export class CppProperties {
                     if (curOffset >= compilerPathStart && curOffset <= compilerPathEnd) {
                         continue;
                     }
-                    let message: string;
+                    let message: string = "";
                     if (!pathExists) {
                         if (curOffset >= forcedIncludeStart && curOffset <= forcedeIncludeEnd
                                 && !path.isAbsolute(resolvedPath)) {
                             continue; // Skip the error, because it could be resolved recursively.
                         }
-                        message = localize('cannot.find2', "Cannot find \"{0}\".", resolvedPath);
+                        // If there are incorrect paths and the first match includes "env:", split the string and check for environment variable.
+                        if (incorrectExpandedPaths.length > 0 && configMatches[0].includes("env:")) {
+                            const splitString = configMatches[0].split("env:");
+                            if (splitString[1]) {
+                                const matchResult = splitString[1].match(/\w+/);
+                                if (matchResult) {
+                                    const envVar = matchResult[0];
+                                    message = localize('cannot.find3', "Cannot find {0} in environment variable: {1}.", incorrectExpandedPaths.map(s => `"${s}"`).join(', '), envVar);
+                                }
+                            }
+                        // If there are incorrect paths but no "env:" in the first match, generate a message indicating the paths cannot be found.
```

Comment:
Can you move this comment after the `} else {`?

Reply:
Umm...but the comment doesn't match what the "else" condition 100%. Should the `else` be an `else if`?

Reply:
Oh, the comment and else seems fine.

Reply:
@browntarik Was this resolved? Is the comment located at the correct spot?

Reply:
No, the comment should be in the block that it describes.

Reply:
@bobbrow I don't understand, are you saying the comment is located at the correct spot or not?

Reply:
It says `generate a message indicating the paths cannot be found.` and then no code comes after it.

Reply:
@sean-mcmanus I was answering your question that the comment was not at the correct spot and it was not resolved.
```ts
if (condition) {
  [code]

// Comment about `else` case should not be here because it's in the `if` block
} else {
  // Comment about `else` case should be here inside the `else` block.
}